File: service-notification/Modeles/dto/FeedBackDTO.cs
using System.Text.Json.Serialization;

namespace service_notification.Modeles.dto
{
    internal class FeedBackDTO
    {
        [JsonPropertyName("id")]
        public int Id { get; set; }

        [JsonPropertyName("idUtilisateur")]
        public int IdUtilisateur { get; set; }

        [JsonPropertyName("sujet")]
        public string Sujet { get; set; }

        [JsonPropertyName("message")]
        public string Message { get; set; }
    }
}

File: service-jeu/src/main/java/m2/miage/interop/servicejeu/dto/FeedBackDTO.java
package m2.miage.interop.servicejeu.dto;

import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class FeedBackDTO {

    long id;

    long idUtilisateur;

    @Size(min = 2, message = "Le message doit contenir au moins 2 caractères")
    String message;

    String sujet;

    public FeedBackDTO(long id, String message, String sujet) {}
}
